pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -8,6 +8,7 @@ import (
 type InfinispanSpec struct {
 	Size  int32  `json:"size"`
 	Image string `json:"image"`
+	JGroupsStack string `json:"jgroups-stack"`
```

Comment:
Hmm, does adding this mean the user can configure the jgroups-stack to use? If so I'd leave it out, the user should not touch this.

Reply:
@galderz you mean there's no gain for the user to choose an alternative discovery protocol?... sounds

Reply:
Yeah, I think that's fine.

Reply:
ok, removed